src/containers/Drawer.tsx

```diff
@@ -108,31 +157,51 @@ export const Drawer = () => {
                   type="text"
                   placeholder="Signaling path"
                   className="input input-bordered w-full max-w-xs"
-                  value={signalingPath || ""}
+                  value={path || ""}
                   onChange={(event) => {
-                    setSignalingPath(event.target.value);
+                    setPath(event.target.value);
                   }}
                 />
               </div>
               <button
                 className="btn btn-sm btn-accent m-1"
                 onClick={() => {
                   setServerToken(DEFAULT_TOKEN);
-                  setSignalingHost(DEFAULT_HOST);
-                  setSignalingPath(DEFAULT_PATH);
-                  setSignalingProtocol(DEFAULT_PROTOCOL);
+                  setHost(DEFAULT_HOST);
+                  setPath(DEFAULT_PATH);
+                  setProtocol(DEFAULT_PROTOCOL);
                 }}
               >
                 Restore default
               </button>
+              <button
+                className="btn btn-sm btn-accent m-1"
+                onClick={() => {
+                  console.log(jellyfishServers.keys());
+                  if (host && protocol && path && serverToken) {
+                    setJellyfishServers(
```

Comment:
It would be better to disable Connect to server when host, protocol, path, and serverToken are empty. This would provide the user with information about what is happening.